sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/event_processor.py

```diff
@@ -114,22 +114,13 @@ def __repr__(self):
         return "EventProcessor: id {}".format(self._id)
 
     def _process_error(self, partition_context, err):
-        _LOGGER.warning(
```

Comment:
So you mean to log warnnings when exception happens and don't warn again in process_error?

Reply:
Yup, that way we can log a context-specific error message rather than a generic one.